python/cuml/cuml/accel/tests/scikit-learn/run-tests.sh

```diff
@@ -11,17 +11,8 @@
 
 set -eu
 
-# Get CUDA version
-CUDA_VERSION=$(nvcc --version | grep "release" | awk '{print $5}' | cut -d',' -f1)
-
 # Base arguments
-PYTEST_ARGS="-p cuml.accel --pyargs sklearn --xfail-list=\"$(dirname "$0")/xfail-list.yaml\""
-
-# Skip sequential tests for CUDA 11.x until
-# https://github.com/rapidsai/cuml/issues/6622 is resolved
-if [[ "$CUDA_VERSION" == "11"* ]]; then
-    PYTEST_ARGS="$PYTEST_ARGS -k \"not test_sequential\""
-fi
```

Comment:
Removing this is technically unrelated, but I guess fine to clean up as part of this PR?

Reply:
yeah, I thought I'd add it in since the CUDA 11 removal PR is already in